internal/utils/utils.go

```diff
@@ -684,3 +684,21 @@ func GetDockerNamespaceEnvVar(fallback string) string {
 	}
 	return fallback
 }
+
+// StartTransaction returns a new Transaction with the specified
+// name and type, with the start time set to the current time and
+// with the context if TRACEPARENT environment variable is set.
+// This is equivalent to calling apm.DefaultTracer.StartTransaction
+// if no TRACEPARENT environment variable otherwise
+// apm.DefaultTracer.StartTransactionOptions
+func StartTransaction(name, transactionType string) *apm.Transaction {
```

Comment:
We are trying to get rid of utils, as it creates coupling between multiple packages. I'd see convenient to move this to a single `apm.go` file at the `./internal` root directory, so that we could create an aliased import:

```go
import apme2e "github.com/elastic/e2e-testing/internal"

...

		suiteTx = apme2e.StartTransaction("Tear Down ${CAPITAL_SUITE}", "test.suite")
```